From a669059bd45a5d6fb68c02f72aab8a5c8b6a7559 Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Thu, 6 Oct 2022 00:31:03 +0530
Subject: [PATCH 12/30] Launcher3: Update lens icon in hotseat QSB

From T pixel launcher.

Change-Id: Ifc186989202a69289d3ecdbf532cb44b6521c512
[neobuddy89: Keep icon size aligned with rest of icons in QSB]

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 res/drawable/ic_lens_color.xml | 28 ++++++++++++----------------
 1 file changed, 12 insertions(+), 16 deletions(-)

diff --git a/res/drawable/ic_lens_color.xml b/res/drawable/ic_lens_color.xml
index 6a04970170..3f1b8e98e7 100644
--- a/res/drawable/ic_lens_color.xml
+++ b/res/drawable/ic_lens_color.xml
@@ -1,26 +1,22 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
+    android:theme="@style/QsbIconTint"
     android:height="24dp"
-    android:viewportWidth="24"
-    android:viewportHeight="24">
-    <path
-        android:name="qsbIconTintTertiary"
-        android:fillColor="#FBBC05"
-        android:pathData="M16,5C17.66,5 19,6.34 19,8V10H21V8C21,5.24 18.76,3 16,3H14V5H16Z" />
+    android:width="24dp"
+    android:viewportWidth="192"
+    android:viewportHeight="192">
     <path
-        android:name="qsbIconTintPrimary"
         android:fillColor="#4285F4"
-        android:pathData="M8,19C6.34,19 5,17.66 5,16V14H3V16C3,18.76 5.24,21 8,21H10V19H8Z" />
+        android:pathData="M68,104.15 A28,28 0 1,1 124,104.15 A28,28 0 1,1 68,104.15" />
+    <path
+        android:fillColor="#EA4335"
+        android:pathData="M160,72v40.15V136c0,1.69-0.34,3.29-0.82,4.82v0v0c-1.57,4.92-5.43,8.78-10.35,10.35h0v0 c-1.53,0.49-3.13,0.82-4.82,0.82H66l16,16h50h12c4.42,0,8.63-0.9,12.46-2.51c3.83-1.62,7.28-3.96,10.17-6.86 c1.45-1.45,2.76-3.03,3.91-4.74c2.3-3.4,3.96-7.28,4.81-11.44c0.43-2.08,0.65-4.24,0.65-6.45v-12V96.15V84l-6-19l-10.82,2.18 C159.66,68.71,160,70.31,160,72z" />
     <path
-        android:name="qsbIconTintPrimary2"
         android:fillColor="#4285F4"
-        android:pathData="M8.5,12C8.4976,11.2595 8.7304,10.537 9.1645,9.937C9.5986,9.3371 10.2123,8.8901 10.9164,8.6609C11.6205,8.4316 12.3795,8.4316 13.0836,8.6609C13.7878,8.8901 14.4014,9.3371 14.8355,9.937C15.2696,10.537 15.5024,11.2595 15.5,12C15.5024,12.7405 15.2696,13.463 14.8355,14.063C14.4014,14.6629 13.7878,15.1099 13.0836,15.3391C12.3795,15.5684 11.6205,15.5684 10.9164,15.3391C10.2123,15.1099 9.5986,14.6629 9.1645,14.063C8.7304,13.463 8.4976,12.7405 8.5,12Z" />
+        android:pathData="M32,72c0-1.69,0.34-3.29,0.82-4.82c1.57-4.92,5.43-8.78,10.35-10.35C44.71,56.34,46.31,56,48,56 h96c1.69,0,3.29,0.34,4.82,0.82c0,0,0,0,0,0L149,45l-17-5l-16-16h-13.44H96h-6.56H76L60,40H48c-17.67,0-32,14.33-32,32v12v20l16,16 V72z" />
     <path
-        android:name="qsbIconTintSecondary"
         android:fillColor="#34A853"
-        android:pathData="M16,18C16,17.4697 16.2109,16.9608 16.5858,16.5858C16.9608,16.2109 17.4697,16 18,16C18.5303,16 19.0392,16.2109 19.4142,16.5858C19.7891,16.9608 20,17.4697 20,18C20,18.5303 19.7891,19.0392 19.4142,19.4142C19.0392,19.7891 18.5303,20 18,20C17.4697,20 16.9608,19.7891 16.5858,19.4142C16.2109,19.0392 16,18.5303 16,18Z" />
+        android:pathData="M144,40h-12l16.83,16.83c1.23,0.39,2.39,0.93,3.47,1.59c2.16,1.32,3.97,3.13,5.29,5.29 c0.66,1.08,1.2,2.24,1.59,3.47v0L176,84V72C176,54.33,161.67,40,144,40z"/>
     <path
-        android:name="qsbIconTintQuaternary"
-        android:fillColor="#EA4335"
-        android:pathData="M8,5C6.34,5 5,6.34 5,8V10H3V8C3,5.24 5.24,3 8,3H10V5H8Z" />
+        android:fillColor="#FBBC05"
+        android:pathData="M48,168h39.89l-16-16H48c-8.82,0-16-7.18-16-16v-23.89l-16-16V136C16,153.67,30.33,168,48,168z"/>
 </vector>
-- 
2.52.0

